main.py

```diff
@@ -8,70 +8,75 @@
 def main():
 
     # IP / DHCP
-    ip = getIPv4()
+    ip = get_ipv4()
     print('IP:           %s' % ip)
     if ip.startswith('169.'):
         print('No DHCP')
         return
 
     # Gateway
-    gatewayIP = getGatewayIP()
-    if gatewayIP == None:
+    gateway_ip = get_gateway_ip()
+    if gateway_ip is None:
         print('Gateway:      NONE')
     else:
-        print('Gateway:      %s' % gatewayIP)
-        print('Gateway ping: %s' % ping(gatewayIP))
+        print('Gateway:      %s' % gateway_ip)
+        print('Gateway ping: %s' % ping(gateway_ip))
 
-    # sendHTTPRequest(gatewayIP)
+    # send_http_request(gateway_ip)
     # return
 
-
     # Teh Internets
-    googleDNSPing = ping('8.8.8.8')
-    print('8.8.8.8 ping: %s' % googleDNSPing)
+    google_dns_ping = ping('8.8.8.8')
+    print('8.8.8.8 ping: %s' % google_dns_ping)
 
-    # if googleDNSPing != 'Down':
+    # if google_dns_ping != 'Down':
     #     return
 
     # NOTE: algo5 does not respond to pings
     # vpnHostPing = ping('138.197.186.179')
     # print('VPN ping:     %s' % vpnHostPing)
 
+    print('')
+    print('Trying to find captive portal... [ALPHA]')
+    print('')
```

Comment:
What's the benefit of this? Also this doesn't replace `/r`

Reply:
Just a matter of preference I guess -- changed it back.

Also TIL --> https://stackoverflow.com/questions/1761051/difference-between-n-and-r

```
The only reason to use \r is if you're writing to a character terminal 
(or more likely a "console window" emulating it) and want the next line you write 
to overwrite the last one you just wrote (sometimes used for goofy "ascii animation" 
effects of e.g. progress bars)
```